clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -0,0 +1,124 @@
+                                  help='Path to the tsv file.')
+        clinica_comp.add_argument("extract_method",
+                help='''Format of the extracted features. Three options:
+                'image' to convert to PyTorch tensor the complete 3D image,
+                'patch' to extract 3D volumetric patches and 
+                'slice' to extract 2D slices from the image.
+                By default the features are extracted from the cropped image.''',
+                choices=['image', 'slice', 'patch'], default='image'
+                )
+
+        # group_id can be used by certain pipelines when some operations are performed at the group level
+        # (for example, generation of a template in pipeline t1-volume)
+        # clinica_comp.add_argument("group_id",
+        #                           help='User-defined identifier for the provided group of subjects.')
+
+        # Add your own pipeline command line arguments here to be used in the
+        # method below. Example below:
+        optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+        optional_slice = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+        optional_patch = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+
+        optional_slice.title="%sPipeline options if you chose ‘slice’ extraction%s" % (Fore.BLUE, Fore.RESET)
+        optional_patch.title="%sPipeline options if you chose ‘patch’ extraction%s" % (Fore.BLUE, Fore.RESET)
+
+        optional.add_argument('-uui', '--use_uncropped_image',
+                              help='''Use the uncropped image instead of the
+                              cropped image generated by t1-linear.''',
+                              default=False, action="store_true"
+                              )
+
+        optional_patch.add_argument(
+                '-ps', '--patch_size',
+                help='''Patch size (only for 'patch' extraction) e.g: --patch_size 50''',
+                type=int, default=50
+                )
+        optional_patch.add_argument(
+                '-ss', '--stride_size',
+                help='''Stride size (only for 'patch' extraction) e.g.: --stride_size 50''',
+                type=int, default=50
+                )
+        optional_slice.add_argument(
+                '-sd', '--slice_direction',
+                help='''Slice direction (only for 'slice' extraction). Three options:
+                '0' -> Sagittal plane,
+                '1' -> Coronal plane or
+                '2' -> Axial plane''',
+                type=int, default=0
+                )
+        optional_slice.add_argument(
+                '-sm', '--slice_mode',
+                help='''Slice mode. Two options: 'rgb' to save the slice in
+                three identical channels, ‘single’ to save the slice in a
+                single channel (default: --slice_mode rgb).''',
+                choices=['rgb', 'single'], default='rgb'
+                )
```

Comment:
Proposition to simplify the use of `_args.add_argument_group`:
```suggestion
        optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
        optional.add_argument('-uui', '--use_uncropped_image',
                              help='''Use the uncropped image instead of the
                              cropped image generated by t1-linear.''',
                              default=False, action="store_true"
                              )

        optional_patch = self._args.add_argument_group(
            "%sPipeline options if you chose ‘patch’ extraction%s" % (Fore.BLUE, Fore.RESET)
        )
        optional_patch.add_argument(
                '-ps', '--patch_size',
                help='''Patch size (default: --patch_size 50).''',
                type=int, default=50
                )
        optional_patch.add_argument(
                '-ss', '--stride_size',
                help='''Stride size (default: --stride_size 50).''',
                type=int, default=50
                )

        optional_slice = self._args.add_argument_group(
            "%sPipeline options if you chose ‘slice’ extraction%s" % (Fore.BLUE, Fore.RESET)
        )
        optional_slice.add_argument(
                '-sd', '--slice_direction',
                help='''Slice direction. Three options:
                '0' -> Sagittal plane,
                '1' -> Coronal plane or
                '2' -> Axial plane
                (default: sagittal plane i.e. --slice_direction 0)''',
                type=int, default=0
                )
        optional_slice.add_argument(
                '-sm', '--slice_mode',
                help='''Slice mode. Two options: 'rgb' to save the slice in
                three identical channels, ‘single’ to save the slice in a
                single channel (default: --slice_mode rgb).''',
                choices=['rgb', 'single'], default='rgb'
                )
```
Don't forget to put default values for each flag